packages/build-utils/src/fs/glob.ts

```diff
@@ -62,7 +62,7 @@ export default async function glob(
         finalPath = path.join(mountpoint, finalPath);
       }
 
-      results[finalPath] = new FileFsRef({ mode: stat.mode, fsPath });
+      results[finalPath.replace(/\\/g, '/')] = new FileFsRef({ mode: stat.mode, fsPath });
```

Comment:
It seems to be possible to create folders with a `\` in the name on MacOS and Linux, so this might have unexpected side effects in those cases, maybe it should only be applied in case the OS is Win?

Reply:
There is something similar a few lines higher:

```js
const fsPath = path.join(options.cwd!, relativePath).replace(/\\/g, '/');
```

So I guess folders with `\` in the name would break in many many places anyway...

Reply:
Lets add a test to confirm this works as expected (we run unit tests on windows/mac/linux)